go/executor/pai.go

```diff
@@ -715,3 +719,16 @@ func (s *paiExecutor) GetTrainStmtFromModel() bool { return false }
 func pickPAILogViewerURL(output string) []string {
 	return reODPSLogURL.FindAllString(output, -1)
 }
+
+func guessError(taskType, output string) error {
```

Comment:
we already give the suggestion to users, so maybe the function name can be `disgnostic`.